### ISSUE TITLE
Minor visual issues in the documentation

### ISSUE THREAD

**Issue by AVHopp** [emdgroup/baybe#195]
This issue is intended to serve as a place where minor visual issues regarding the documentation can be collected. Once enough such have been identified, they will be resolved in a corresponding PR.

Note in particular that this issue should not be used to collect major changes. Basically, if you see something in our documentation and think "Huh, that just looks a bit weird and would benefit from reformatting", then this is the place to just put it 😄

**Comment by AVHopp** [comment]
There is a cut-off in the table describing the numerical parameters (https://emdgroup.github.io/baybe/_autosummary/baybe.parameters.numerical.html)
<img width="932" alt="image" src="https://github.com/emdgroup/baybe/assets/84016733/462cc344-7d37-4abf-ac08-5ff15abb6b33">

**Comment by Scienfitz** [comment]
In general, getting rid of progress bar outputs or at least just having one line (to show the user that somethign WAS computing) would be nice. [Here](https://emdgroup.github.io/baybe/examples/Backtesting/impute_mode.html) in addition we have weird formatting jumps too:
![image](https://github.com/emdgroup/baybe/assets/17951239/1a9b5596-4aad-437f-809e-c5d34c72eb9a)

**Comment by AVHopp** [comment]
@Scienfitz the `tqdm` progress bars are already gone, which bars are you referring to?

**Comment by Scienfitz** [comment]
@AVHopp thats right I thought I still saw progress bars somewhere but it seems theyre gone and I probably confused some of the cache notifications with them

**Comment by Scienfitz** [comment]
This table in the searchspace userguide looks badly formatted in light mode: 
![image](https://github.com/emdgroup/baybe/assets/17951239/1948103c-c761-403f-8377-ca6d411b95a1)
simply fixed by removing the code formatting in table headers

**Comment by brandon-holt** [comment]
> @Scienfitz the `tqdm` progress bars are already gone, which bars are you referring to?

Would definitely be nice to have some indication of how far along the search is though! 

Curious why this was removed?

**Comment by Scienfitz** [comment]
@brandon-holt this Issue is just about the documentation, where progress bars have been removed. The progress bars when using our simulation module should still be there

**Comment by brandon-holt** [comment]
@Scienfitz In the Simulation section, the section for simulate_experiment contains a codeblock for simulate scenarios:
https://emdgroup.github.io/baybe/userguide/simulation.html

Also, the codeblocks used don't make certain aspects of usage clear. For example, for the campaign variable, do we need to define a searchspace, or will the searchspace be automatically generated from initial_data? 

Or, if we use the DataFrame option for the lookup variable, is that not the same dataframe as for initial_data?

**Comment by Scienfitz** [comment]
@brandon-holt thanks for spotting that! technically your points are not visual but about a bug and content suggestions, so we should not discuss these in this thread.

Would you be open to make s small PR fixing the code snippet? The respective .md file is in `docs/userguide/simulation.md` . You could also propose text changes that make clearer what you seek. 

As for the searchspace question: Nothing is inferred from `initial_data`, is there anything specific that gave you that impression?

**Comment by brandon-holt** [comment]
@Scienfitz Just made the PR!

Also, disregard my question about searchspace/initial_data I understand now!

**Comment by AdrianSosic** [comment]
deprecated strategies namespaces needs to be put on the blacklist
<img width="281" alt="image" src="https://github.com/emdgroup/baybe/assets/23265127/c25ddef8-c6bf-41d8-89c1-86fe3ba05937">

**Comment by AdrianSosic** [comment]
Toc depth of 2 is fine for the examples, but the user guide should have one level less
<img width="451" alt="image" src="https://github.com/emdgroup/baybe/assets/23265127/157e8c2b-1ac3-4492-b794-8067f4be0cb6">

**Comment by AdrianSosic** [comment]
This is what the readme page on Github looks like in dark mode
<img width="269" alt="image" src="https://github.com/emdgroup/baybe/assets/23265127/eec1d916-4c40-4e3d-a1f4-a3d814a826d1">

**Comment by Scienfitz** [comment]
autodoc page badly cutoff to the right:
![image](https://github.com/emdgroup/baybe/assets/17951239/3e257362-4470-44a5-bfea-79a3b33c272f)

**Comment by Scienfitz** [comment]
building the doc now has ugly warnigns all over covering the progress bars in case examples are built
![image](https://github.com/user-attachments/assets/973ddfb4-1eba-4fcf-b0f3-461d786348aa)

**Comment by AVHopp** [comment]
Since the question came up in #347 : Some of the errors mentioned here are fixed in #347 , some things I do not know how to fix (like changing the name of `baybe` in the TOC, I already tried that several times).
Regarding the other things here: Depending on the urgency, I can take care of these in an upcoming PR and then close this issue and we start another one once we accumulate more stuff. What is your opinion here?

**Comment by Scienfitz** [comment]
maybe I dont get the intention behind the `--f` flag for docs building. But even when using it I get errors related to timed out linkchecks etc

Command:
```terminal
SMOKE_TEST=ja tox -e docs-py310 -- -f
```

Result:
```terminal
(_autosummary/baybe.surrogates.gaussian_process.presets.edbo.EDBOKernelFactory: line    6) timeout   https://doi.org/10.1038/s41586-021-03213-y - HTTPSConnectionPool(host='idp.nature.com', port=443): Read timed out. (read timeout=30)
The name of the builder is: linkcheckbuild finished with problems, 146 warnings.
```
